daisy_workflows/windows_upgrade/reboot.wf.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
reboot_instance.wf.json

Reply:
I don't get why that is better since there isn't anything else that can be rebooted